required/latex-lab/math-tagging-examples/math-17-alt+mathmlAF+texAF.tlg

```diff
@@ -1,3 +1,3 @@
 ***************
-Compilation 1 of test file completed with exit status 0
-Compilation 2 of test file completed with exit status 0
+Compilation 1 of test file completed with exit status 1
+Compilation 2 of test file completed with exit status 1
```

Comment:
why do we have a change in exit code here?

Reply:
I cleaned up and added \START but forgot regression-test (these files are not part of the testsuite, only meant as examples, so github didn't catch the change).